### ISSUE TITLE
error on start fauxton after npm install -g

### ISSUE THREAD

**Issue by gabrielmancini** [apache/couchdb-fauxton#903]
```
fauxton

 ______                        _
|  ____|                      | |
| |__    __ _   _   _  __  __ | |_    ___    _ __
|  __|  / _` | | | | | \ \/ / | __|  / _ \  | '_ \
| |    | (_| | | |_| |  >  <  | |_  | (_) | | | | |
|_|     \__,_|  \__,_| /_/\_\  \__|  \___/  |_| |_|

Listening on 8000
Could not locate /Users/gabrielmancini/.nvm/versions/node/v6.9.4/lib/node_modules/fauxton/dist/release/index.html
```

**Comment by BenAychh** [comment]
Same here, had to go back to 1.1.10

**Comment by michellephung** [comment]
hey
@gabrielmancini @BenAychh 

I try to publish a new version at the beginning of each month. 
I just pushed a new version to npm, can you guys try again to see if 1.1.12 works for you, if you have time?

**Comment by BenAychh** [comment]
@michellephung  That did it, thanks :)

**Comment by Antonio-Maranhao** [comment]
Closing since the problem was solved in 1.1.12.
Please reopen if that's not the case.
